regolith/schemas.py

```diff
@@ -3798,17 +3872,22 @@
                                                 "in these groups their names or id's can be added explicitly to the list",
                                  "required": False,
                                  "type": "list"},
-                    "status": {"description": f"status of the milestone. "
-                                              f"Allowed values are {', '.join(PROJECTUM_STATUS)}",
+                    "status": {"description": "status of the milestone. "
```

Comment:
see comment above.  Don't hard code values in here, it makes the code much harder to maintain.

Reply:
Done